conf/m3c.config

```diff
@@ -16,9 +16,29 @@ process {
         time: 14.d
     ]
     executor         = 'slurm'
-    queue            = { task.time > 23.h ? 'cpu3-long' : (task.memory > 460.GB || task.cpus > 64 ? 'cpu2-hm' : 'cpu1') }
     scratch          = 'true'
     containerOptions = '--bind $TMPDIR'
+    queue = {
+        if (task.label && task.label.toString().contains('gpu')) {
+            return 'gpu-a30'
+        } else if (task.time > 24.h) {
```

Comment:
Why did you change it? could this be a problem for tasks taking 24h?

Reply:
if it get's only killed when > 24h, then it makes sense changing it as you did (not sure how the exact setting was)